sdk/dapp-kit/package.json

```diff
@@ -73,7 +74,8 @@
 		"@mysten/sui.js": "workspace:*"
 	},
 	"peerDependencies": {
-		"react": "*"
+		"react": "*",
+		"@tanstack/react-query": "^4.0.0"
```

Comment:
So this is the approach we landed on?

Reply:
I didn't actually change this after our last conversation... already had this on the branch I was working off of... I don't remember where we landed... but I think this seems right?

Setup would just always say something like:

## Install
```bash
npm install --save @mysten/dapp-kit @tanstack/react-query
```

Reply:
Yeah that's fine with me!